sklego/neighbors.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.neighbors import KernelDensity
+from sklearn.utils import check_X_y
+from sklearn.utils.multiclass import unique_labels
+from sklearn.utils.validation import check_is_fitted, check_array, FLOAT_DTYPES
+
+
+# noinspection PyPep8Naming,PyAttributeOutsideInit
```

Comment:
what is this comment for?

Reply:
it suppresses automatic lint checkers in PyCharm since `sklearn` uses a few unpythonic practices. It won't be there in the final commit.